### ISSUE TITLE
Form elements should have name property

### ISSUE THREAD

**Issue by simonh1000** [hperrin/svelte-material-ui#643]
**Is your feature request related to a problem? Please describe.**
Svelte puts a lot of emphasis on writing forms using html form syntax. That requires `name` properties to create the right data for for `<form method="POST"...>` It doesn't look as though those can be added 

**Describe the solution you'd like**
a name field on Radio, TextField, ... that is put onto the form element itself

**Describe alternatives you've considered**
I'm going to try using the current reactive values and attaching that to a hidden field

**Additional context**

**Comment by TjenWellens** [comment]
> **Describe alternatives you've considered** I'm going to try using the current reactive values and attaching that to a hidden field
Thanks for this workaround hack! I was stuck with how to send form data.

But the form elements really should have a `name` property!

**Comment by TjenWellens** [comment]
They apparently do have a `name property`!
#464 : it's set on the `label` instead of the `input`

> The behavior you're after can be implemented with `input$name={name}`, or for a bind, `bind:input$name={name}`.
src: https://github.com/hperrin/svelte-material-ui/issues/464#issuecomment-1288216140

**Comment by hperrin** [comment]
Yes. Since the `name` has no functionality outside of the `input` (unlike `disabled`, which adds styling to the surrounding elements), it doesn't have a "proxy" attribute on the outside component. So, it should be added using the internal component syntax (`input$name="whatever"`).
